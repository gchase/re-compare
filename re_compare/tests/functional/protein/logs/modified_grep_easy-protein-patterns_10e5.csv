regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,1183,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,1183,"[-1, -1]"
DHKNLD.,first_match,0,1994,"[-1, -1]"
DHKNLD.,all_matches,0,1994,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,1874,"[98918, 98923]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,1874,"[98918, 98923]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,12,"[99267, 99272]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,1886,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,1154,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,1154,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,883,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,1,883,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,2,3,"[3, 3]"
G|S|T|A|D|N|E|K|R,all_matches,2,886,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,2594,"[49972, 49977]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,2594,"[49972, 49977]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,39,"[50385, 50390]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,2633,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,1099,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,1,1099,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,2,7,"[440, 441]"
(N|S.TNH.Y)(F|W),all_matches,2,1106,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,1592,"[11708, 11712]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,1592,"[11708, 11712]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,465,"[58928, 58932]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,2057,"[-1, -1]"
P..L...KWE.,first_match,0,1114,"[-1, -1]"
P..L...KWE.,all_matches,0,1114,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,1868,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,1868,"[-1, -1]"
