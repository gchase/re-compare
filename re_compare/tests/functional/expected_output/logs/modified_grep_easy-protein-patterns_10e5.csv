regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,2035,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,2035,"[-1, -1]"
DHKNLD.,first_match,0,943,"[-1, -1]"
DHKNLD.,all_matches,0,943,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,3532,"[98918, 98923]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,3532,"[98918, 98923]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,13,"[99267, 99272]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,3545,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,1213,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,1213,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,2644,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,1,2644,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,2,5,"[3, 3]"
G|S|T|A|D|N|E|K|R,all_matches,2,2649,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,2300,"[49972, 49977]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,2300,"[49972, 49977]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,17,"[50385, 50390]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,2317,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,1563,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,1,1563,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,2,12,"[440, 441]"
(N|S.TNH.Y)(F|W),all_matches,2,1575,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,1861,"[11708, 11712]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,1861,"[11708, 11712]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,811,"[58928, 58932]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,2672,"[-1, -1]"
P..L...KWE.,first_match,0,1102,"[-1, -1]"
P..L...KWE.,all_matches,0,1102,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,2051,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,2051,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,1376,"[829, 837]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,1376,"[829, 837]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,2,13,"[995, 1003]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,2,1389,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,1,1676,"[41350, 41356]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),consecutive,1,1676,"[41350, 41356]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,1,2293,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,1102,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,1102,"[-1, -1]"
A|G....G,first_match,1,1007,"[14, 14]"
A|G....G,consecutive,1,1007,"[14, 14]"
A|G....G,consecutive,2,3,"[18, 18]"
A|G....G,all_matches,2,1010,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,1,6243,"[57781, 57785]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,1,6243,"[57781, 57785]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,2,23,"[58000, 58004]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,2,6266,"[-1, -1]"
